docs/integrations/connector-health.md

```diff
@@ -18,6 +18,7 @@ Clicking on a status will take you to a summary of past tests that link to the G
 
 | Source | Build Status | Airbyte Certified | Notes |
 | :--- | :--- | :---: | :--- |
+| Amplitude | [![source-amplitude](https://img.shields.io/endpoint?url=https%3A%2F%2Fstatus-api.airbyte.io%2Ftests%2Fsummary%2Fsource-amplitude%2Fbadge.json)](https://status-api.airbyte.io/tests/summary/source-amplitude) |  |  |
```

Comment:
Does this make to call source `check_connection ` method?

Reply:
No, this shows you results of lasts CI Tests